controllers/apply/under10k/fields/project-dates-next.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
+const get = require('lodash/fp/get');
+const moment = require('moment');
+const { oneLine } = require('common-tags');
+
+const Joi = require('../../lib/joi-extensions');
+const { DateField, RadioField } = require('../../lib/field-types');
+
+function getLeadTimeWeeks(country) {
+    const countryLeadTimes = {
+        'england': 18,
+        'northern-ireland': 12,
+        'scotland': 12,
+        'wales': 12,
+    };
+
+    return countryLeadTimes[country] || 18;
+}
+
+module.exports = {
+    _getLeadTimeWeeks: getLeadTimeWeeks, // Exported for tests
+
+    fieldProjectStartDateCheck(locale, data = {}) {
+        const localise = get(locale);
+
+        const projectCountry = get('projectCountry')(data);
+        const supportingCOVID19 = get('supportingCOVID19')(data);
+
+        function options() {
+            const optionAsap = {
+                value: 'asap',
+                label: localise({
+                    en: `As soon as possible`,
+                    cy: `@TODO: i18n`,
+                }),
+            };
+
+            const optionExactDate = {
+                value: 'exact-date',
+                label: localise({
+                    en: `Enter an exact date`,
+                    cy: `@TODO: i18n`,
+                }),
+            };
+
+            if (projectCountry === 'england' || supportingCOVID19 === 'yes') {
+                optionExactDate.attributes = { disabled: 'disabled' };
+            }
+
+            return [optionAsap, optionExactDate];
+        }
+
+        return new RadioField({
+            locale: locale,
+            name: 'projectStartDateCheck',
+            label: localise({
+                en: `When would you like to get the money if you are awarded?`,
```

Comment:
I know this is yet to be finalised but I think the wording/terminology (especially "awarded") is a bit clunky here